src/vmspawn/vmspawn-scope.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include <stdbool.h>
+
+#include "sd-bus.h"
```

Comment:
as elsewhere, please keep sd-*.h includes separated from the rest via an empty line